shared/profile/prove-enter-username-container.js

```diff
@@ -0,0 +1,38 @@
+// @flow
```

Comment:
i named this -container as these classes are normally called -render but they weren't named that.